src/app/layout.tsx

```diff
@@ -8,9 +8,9 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en" data-mode="light">
-      <body>
+      <body className="h-screen">
```

Comment:
Don't think we need the added classes here. Looks good otherwise!

Reply:
If there is no class in here the spinner isn't in the middle but on the top